Tests/BaseObjectTests.swift

```diff
@@ -1,13 +1,15 @@
 import Quick
 import Nimble
-import Clappr
+@testable import Clappr
 
 class BaseObjectTests: QuickSpec {
 
+    class ConcreteBaseObject: BaseObject { }
+
     override func spec() {
         describe("BaseObject") {
```

Comment:
According to the [Project conventions](https://github.com/globoi/steve/wiki/Test-Conventions), we should add the . prefix on the Class Name